common/frame.go

```diff
@@ -737,24 +717,28 @@ func (f *Frame) ChildFrames() []api.Frame {
 func (f *Frame) Click(selector string, opts goja.Value) {
 	f.log.Debugf("Frame:Click", "fid:%s furl:%q sel:%q", f.ID(), f.URL(), selector)
 
-	parsedOpts := NewFrameClickOptions(f.defaultTimeout())
-	err := parsedOpts.Parse(f.ctx, opts)
-	if err != nil {
+	popts := NewFrameClickOptions(f.defaultTimeout())
+	if err := popts.Parse(f.ctx, opts); err != nil {
+		k6Throw(f.ctx, "%w", err)
+	}
+	if err := f.click(selector, popts); err != nil {
 		k6Throw(f.ctx, "%w", err)
 	}
+	applySlowMo(f.ctx)
+}
 
-	fn := func(apiCtx context.Context, handle *ElementHandle, p *Position) (interface{}, error) {
-		return nil, handle.click(p, parsedOpts.ToMouseClickOptions())
+func (f *Frame) click(selector string, opts *FrameClickOptions) error {
```

Comment:
If you exported this method (and renamed it too), couldn't you call this method from locator's `click` method?

Reply:
What do you have in mind? Tell me more :)

Reply:
Oh, never mind, I've just noticed that the call from `locators.go` is already calling this unexported method `return l.frame.click(l.selector, opts)`.

Reply:
I'll tell you more about it in the next meeting :)